components/automate-ui/src/app/pages/policy/details/policy-details.component.html

```diff
@@ -40,23 +40,25 @@
         <div *ngIf="(members$ | async)?.length === 0" id="getting-started">Add the first members to get started!</div>
         <chef-toolbar>
           <chef-button primary
-            [routerLink]="['/settings', 'policies', policy.id, 'add-members']">Add Members</chef-button>
+            [routerLink]="['/settings', 'policies', policy?.id, 'add-members']">Add Members</chef-button>
         </chef-toolbar>
         <chef-table *ngIf="(members$ | async)?.length > 0">
           <chef-thead>
             <chef-tr>
-              <chef-th>Name</chef-th>
+              <chef-th>ID</chef-th>
               <chef-th>Type</chef-th>
               <chef-th></chef-th>
             </chef-tr>
           </chef-thead>
           <chef-tbody>
             <chef-tr *ngFor="let member of members$ | async">
-              <!-- TODO (tc) we should link to local users and teams once we switch those APIs to named IDs.
-              can't now since we don't have the GUIDs for them unless we queried them by name for each
-              row somehow. -->
               <chef-td>
-                {{ member.displayName }}
+                <a *ngIf="memberURLs.hasOwnProperty(member.name)" [routerLink]="memberURLs[member.name]" target="_blank">
```

Comment:
i noticed when i opened the first new tab the welcome modal popped up again in the new tab. didn't happen on the second time though.

if we want the default behavior to be `open a new tab` on team/policy member links, let's do a quick inventory and make that consistent for all our pages. i noticed clicking a user from the team membership table doesn't open a new tab.

Reply:
Should definitely make that consistent. Let's take an inventory. I'll add a tech debt card to track down the places the behavior varies and we'll make it consistent!

Reply:
if they are in a user flow already then it should open in a new tab, otherwise, it should not. when we update the add users flow for teams then those links will be fixed to match the flow when adding members to a policy.

i dont think anything else is a user flow like that so no need to open in a new tab... unless im overlooking something.

Reply:
I'll make it not open a new tab in policy-details but open a new tab in the add members modal in that case.